### DISCUSSION TITLE
Is there a way to hit Favorite on a shared album image?

### DISCUSSION THREAD

**Discussion by ToddCrimson** [immich-app/immich#14262] [Q&A] [answered]
Hello, I just setup the server. Great piece of software, thank you.

Need help with following use case.

I have setup a user, and given access to three of my albums.
I need the user to go through the images, in each album, and if the user likes an image, hit the favorite button.
So now it should appear in the favorites tab.
But there is no Favorites button on top. I can see it under my admin user, but the shared user cannot see. They just see the comment button at bottom.

What I'm trying to do is following:

1. User goes through 1500 images.
2. User hits like on 140 images.
3. This way I know which images they want.

Would love some help, without giving the user my admin user details.

Thank you.

**Comment by alextran1502** [comment, marked as answer]
Hello, this is not possible yet. We have plan to make it possible though

**Reply by ToddCrimson** [reply]
ahh ok, thank you. 

Yes it would be an amazing feature as that would help me be able to get input back.

Cheers, and once again thanks for great work.
